test/visibility/private/uses/definesMultiUse.chpl

```diff
@@ -16,7 +16,7 @@ module definesMultiUse {
   }
 
   module Inner3 {
-    var e: int;
+    var ee: int;
```

Comment:
This change really makes me look forward to when Math.e is not included by default

Reply:
Lydia: Since I had the same reaction on multiple of the previous changes, what do you think about the proposal to start teasing Math and AutoMath apart sooner, even though we haven't made final decisions for some of the cases that are sitting on the fence currently?  Moving even just the most obvious things would be a big win, it seems like (and probably not too hard).